test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/K8SequentialExecutor.java

```diff
@@ -171,8 +171,6 @@ private static String getArgs() {
                 "pravegaOperatorVersion",
                 "bookkeeperOperatorVersion",
                 "zookeeperOperatorVersion",
-                "desiredPravegaCMVersion",
```

Comment:
If we are removing configmap version from supported version map, provide support from operators that is removing supported version.

Reply:
done